openapitor/tests/types/input/websocket.json

```diff
@@ -0,0 +1,74 @@
+{
```

Comment:
This was generated from [the official Dropshot example websocket server](https://github.com/oxidecomputer/dropshot/blob/main/dropshot/examples/websocket.rs#L62-L78)